twitter.html

```diff
@@ -0,0 +1,275 @@
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@24,400,0,0" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@24,400,0,0" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@24,400,0,0" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@24,400,0,0" />
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+
+    <div class="container">
+
+        <div class="left-bar">
+
+            <i class="logo"><?xml version="1.0" encoding="UTF-8" standalone="no"?> <svg width="48" height="48" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path d="M0 24C0 10.7452 10.7452 0 24 0C37.2548 0 48 10.7452 48 24C48 37.2548 37.2548 48 24 48C10.7452 48 0 37.2548 0 24Z" fill="#55ACEE"/>
+                <path d="M23.2812 19.5075L23.3316 20.338L22.4922 20.2363C19.4369 19.8465 16.7677 18.5245 14.5013 16.3043L13.3934 15.2027L13.108 16.0162C12.5036 17.8297 12.8897 19.7448 14.1488 21.0328C14.8203 21.7447 14.6692 21.8464 13.5109 21.4227C13.108 21.2871 12.7554 21.1854 12.7219 21.2362C12.6044 21.3549 13.0073 22.8971 13.3262 23.5073C13.7627 24.3547 14.6524 25.1851 15.6261 25.6766L16.4487 26.0664L15.475 26.0834C14.5349 26.0834 14.5013 26.1003 14.6021 26.4562C14.9378 27.5578 16.264 28.7273 17.7413 29.2357L18.7822 29.5916L17.8756 30.1339C16.5326 30.9136 14.9546 31.3542 13.3766 31.3881C12.6211 31.4051 12 31.4728 12 31.5237C12 31.6932 14.0481 32.6423 15.24 33.0151C18.8157 34.1167 23.063 33.6422 26.2526 31.761C28.5189 30.4221 30.7852 27.7612 31.8428 25.1851C32.4136 23.8123 32.9844 21.304 32.9844 20.1007C32.9844 19.3211 33.0347 19.2194 33.9748 18.2873C34.5288 17.7449 35.0492 17.1517 35.15 16.9823C35.3178 16.6603 35.3011 16.6603 34.4449 16.9484C33.018 17.4568 32.8165 17.389 33.5216 16.6264C34.042 16.084 34.6631 15.101 34.6631 14.8129C34.6631 14.7621 34.4113 14.8468 34.1259 14.9993C33.8238 15.1688 33.1523 15.423 32.6486 15.5756L31.7421 15.8637L30.9195 15.3044C30.4663 14.9993 29.8283 14.6604 29.4926 14.5587C28.6364 14.3214 27.327 14.3553 26.5548 14.6265C24.4563 15.3892 23.1301 17.3551 23.2812 19.5075Z" fill="white"/>
+                </svg>
+            </i>
+
+          <nav>
+            <div class="home-logo">
+                <a href="#home"><i class="fa fa-fw fa-home"></i>Home</a>
+            </div>
+
+
+            <div class="explore-logo">
+              <a href="#explore">
+                <i class="fa fa-fw fa-hashtag"></i>Explore
+              </a>
+            </div>
+
+
+            <div class="notifications-logo">
+               <a href="#notifications">
+                <i class="fa fa-fw fa-bell"></i>Notifications
```

Comment:
Woah! What is this? "<i.>"?   I've not used this before. Can you let me know what this is?

Reply:
Icons usually come wrapped in <i> or <span>. I have both because I took them from different sources but now you really got me thinking; which is better to use and how easy can you manipulate them... I'll have a closer look into this.
Good observation !

"Some people feel that `<i>` represents “italic” because once upon a time it did.

Today `<i>` officially stands for “idiomatic”. This change happened with the advent of css and a clear, effective way to separate style from content.

Icons could easily be termed “idiomatic”; therefore, `<i>` is perfectly appropriate for icons.
`<span>` is simply a generic inline element in the same way that `<div>` is a generic block element.

Cases for`<i>` over `<span>` include:
- Declaring an icon an “idiom” is more specific then just a “generic inline element”
- An icon may be used as a block element, and it is more appropriate to declare `i {display: block;} ` than `<span>`, which by definition is not a block element."     ---    a good explanation I found on google